pkg/template/api/v1/types.go

```diff
@@ -14,15 +14,23 @@ type Template struct {
 	// Standard object's metadata.
 	kapi.ObjectMeta `json:"metadata,omitempty"`
 
-	// Objects is an array of objects to include in this template. Required.
+	// message is an optional instructional message that will
+	// be displayed when this template is instantiated.
```

Comment:
- Instantiated - my initial wording was "processed", but it was suggested that the only reason we process templates is to instantiate the underlying objects in etcd.
- Correct, the present implementation does not attempt to output the message during "oc process" (or create). Only targeting new-app and the web console with @jwforres 's help.